src/main/frontend/extensions/tldraw.cljs

```diff
@@ -23,7 +23,7 @@
 
 (rum/defc page-cp
   [props]
-  (page/page {:page-name (gobj/get props "pageName") :whiteboard? true}))
+  (page/page {:page-name (model/get-redirect-page-name (gobj/get props "pageName")) :whiteboard? true}))
```

Comment:
Is there a more appropriate method for this case?

Reply:
This is to fix the alias issue, right? It's ok here. 
The implementation of the fn `model/get-redirect-page-name` is obscure, though.